modules/account-lib/src/coin/hbar/walletInitializationBuilder.ts

```diff
@@ -1,6 +1,5 @@
 import { BaseCoin as CoinConfig } from '@bitgo/statics/dist/src/base';
 import * as hex from '@stablelib/hex';
```

Comment:
I guess we should remove this library right? Since it was removed from the KeyPair PR